_data/Diplomatie/Diplomatie.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Diplomatie
+institution: University Mcgill
+profile_pic: Diplomatie.jpg
+quote: “It is wrong always, everywhere, and for anyone, to believe anything upon insufficient evidence.” - William Kingdon Clifford
+github_user: diplomatie
```

Comment:
The username you've written here is incorrect.

The first letter is "d", while your username begins with a "D". Ensure that it should exactly match.